
From: Anton Blanchard <anton@samba.org>

From: Dave Hansen <haveblue@us.ibm.com>

This is a helper function that a few architectures already have.  This just
copies the i386 implementation to ppc64.

Signed-off-by: Dave Hansen <haveblue@us.ibm.com>
Signed-off-by: Anton Blanchard <anton@samba.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-ppc64/page.h |    1 +
 1 files changed, 1 insertion(+)

diff -puN include/asm-ppc64/page.h~ppc64-add-a-pfn_to_kaddr-function include/asm-ppc64/page.h
--- 25/include/asm-ppc64/page.h~ppc64-add-a-pfn_to_kaddr-function	2004-08-30 00:24:01.928627256 -0700
+++ 25-akpm/include/asm-ppc64/page.h	2004-08-30 00:24:01.933626496 -0700
@@ -240,6 +240,7 @@ extern int page_is_ram(unsigned long phy
 #endif
 
 #define virt_to_page(kaddr)	pfn_to_page(__pa(kaddr) >> PAGE_SHIFT)
+#define pfn_to_kaddr(pfn)	__va((pfn) << PAGE_SHIFT)
 
 #define virt_addr_valid(kaddr)	pfn_valid(__pa(kaddr) >> PAGE_SHIFT)
 
_
